### PR TITLE
Feature/30 copy readme

### LINKED ISSUE #30: Make ReadMe.md copy into nuget package on build.
Currently, there are two copies of ReadMe.md in the repo, one at the root, and one in the core project for the nuget package.

Delete the one in the core project from git, add to .gitignore, and add a copy command to the core library csproj to copy it on build.

### PR DESCRIPTION
**Pull request by TimPurdum**
Closes #30 
Prevents having to maintain two copies of ReadMe
On build, copies ReadMe into location for the Nuget Packaging.

### CODE REVIEW

File: src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj [hunk 1]
```diff
@@ -27,46 +27,51 @@
...
     </ItemGroup>
-    
-    <Target Name="NPM Install" AfterTargets="PreBuildEvent">
-        <Exec Command="npm install" />
+
+    <Target Name="Copy Readme" AfterTargets="PreBuildEvent">
+        <Exec ContinueOnError="true" Command="robocopy ../../ ./ ReadMe.md /it"/>
+    </Target>
+
+    <Target Name="NPM Install" AfterTargets="Copy Readme">
```
Comment (seahro) [inline src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj:60]: Looks like a reformatted file with a single change (NPM Intall?)

File: .gitignore [hunk 2]
```diff
@@ -359,4 +359,5 @@ MigrationBackup/
 */dymaptic.GeoBlazor.Core.Test/ScreenShots/**/*
 /src/.vs/**/*
 /src/.idea/**/*
-/src/dymaptic.GeoBlazor.Core/wwwroot/js/*
\ No newline at end of file
+/src/dymaptic.GeoBlazor.Core/wwwroot/js/*
+/src/dymaptic.GeoBlazor.Core/ReadMe.md
```
Comment (seahro) [inline .gitignore:363]: added readme...looks good

**Review by seahro — APPROVED** [review]
added NPM install reference and readme reference, remainder were reformatting

File: src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj (hunk 1 above)
Comment (TimPurdum) [reply src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj:60]: It's actually the `Copy Readme` command added, and the NPM Install, which was already there, got shoved downward.

**Pull request merged**